pyproject.toml

```diff
@@ -16,17 +16,19 @@ classifiers = [
     "Framework :: Django :: 3.2",
     "Framework :: Django :: 4.0",
     "Framework :: Django :: 4.1",
+    "Framework :: Django :: 5.0",
     "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
     "Programming Language :: Python :: 3 :: Only",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
 ]
 packages = [{ include = "csp" }]
 
 [tool.poetry.dependencies]
 python = "^3.8"
```

Comment:
Looks like this should be 3.10, not 3.8, unless 3.8 _is_ supported, in which case we should update the tox / classifiers?

Reply:
Looks like support was dropped here https://github.com/yunojuno/django-csp-plus/commit/56fdca52ed574865af1eb620a6b3528a8b5c8377 so this was probably left by accident - will update.